### PR TITLE
chore: CXPUI 45 : Handle redirects for URL when quickstart name changes

### PR DESCRIPTION
**Pull request by dikshab0202**
JIRA: https://newrelic.atlassian.net/browse/CXPUI-45
Description: handle redirects for URLs when a quickstart name changes
Fix: added new redirect json file and createRedirects action to generate any redirects

### CODE REVIEW

**Review by pkosanam — CHANGES_REQUESTED** [review]
Please make sure the indentation on quickstart-redirects.json file is right

Hi @aswanson-nr, request you to kindly review this PR as we have changed the URLs in "to" and "from" 
Thank you.

**Review by aswanson-nr — APPROVED** [review]
Tested the redirect and it works perfectly!

File: gatsby-node.js [hunk 1]
```diff
@@ -19,6 +19,15 @@ exports.createPages = async ({ actions, graphql, reporter }) => {
     }
   `);
 
+  externalRedirects.forEach(({ from, to }) => {
+    createRedirect({
+      fromPath: from,
+      toPath: to,
+      isPermanent: true,
+      redirectInBrowser: true,
```
Comment (aswanson-nr) [inline gatsby-node.js:27]: Let's remove this option, it makes Gatsby use meta redirects to change pages. Which can have a bad impact on SEO results. Just FYI, when working with non-browser redirects and Gatsby, the redirects will not work in development mode and PR previews. What happens is Gatsby Cloud picks up the generated `build/_redirects.json` file and uses that to do server side redirects, which only happens on production domains.
```suggestion
```

**Review by aswanson-nr — CHANGES_REQUESTED** [review]
Noticed one small thing we need to change!

**Comment by dikshab0202** [comment]
> Noticed one small thing we need to change!

Hi @aswanson-nr , updated the commit suggestion in PR. Thank you..

**Review by aswanson-nr — APPROVED** [review]

**Review by pkosanam — APPROVED** [review]
looks good

**Pull request merged**